### PR TITLE
Feature/psa tracker ui

### PR DESCRIPTION
**Pull request by achen2401**
address this story:
https://jira.movember.com/browse/TN-581

psa tracker UI

### CODE REVIEW

File: portal/static/js/psaTracker.js [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+                    var existingItem = this.getExistingItemByDate(newDate);
+                    if (existingItem.length > 0) {
+                        this.newItem.id = existingItem[0].id;
+                    }
+                    this.postData();
+                },
+                getData: function() {
+                    var self = this;
+                    this.loading = true;
+                    this.tnthAjax.getClinical($("#psaTrackerUserId").val(), function(data) {
```
Comment (ivan-c) [inline portal/static/js/psaTracker.js]: Is the current user cached in localstorage anywhere?

File: portal/static/js/psaTracker.js [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+                            }
+                        }
+                        setTimeout(function() {
+                            self.loading = false;
+                        }, 550);
+
+                    });
+                },
+                postData: function() {
+                    var userId = $("#psaTrackerUserId").val();
```
Comment (ivan-c) [inline portal/static/js/psaTracker.js]: Same as above comment

File: portal/templates/psa_tracker.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+{% block head %}
+	{{super()}}
+	 <link rel="stylesheet" href="{{ url_for('static', filename='css/psaTracker.css') }}">
+{% endblock %}
+{% block main %}
+<main id="mainPsaApp">
+    <input type="hidden" id="psaTrackerUserId" value="{{user.id}}"/>
+    <section class="psa-tracker-title">
+        <h2 class="tnth-header heading">{{_("Track Your PSA")}}</h2>
+        {{_("Prostate-specific antigen, or PSA, is a protein produced by cells of the prostate gland.  The PSA test measures the level of PSA in a man's blood.  For this test, a blood sample is sent to a laboratory for analysis.  The results are reported as nanograms of PSA per milliliter (ng/mL) of blood.")}}
```
Comment (ivan-c) [inline portal/templates/psa_tracker.html]: Looks like there's a few extra spaces after the periods, not sure if that's intentional or not...

**Comment by ivan** [comment]
@achen2401 when you @ivan, the wrong Ivan (me) gets an email, this has happened six times 
screenshot_2018-04-11_19-41-39

**Pull request merged**